src/app/client/src/app/modules/public/components/public-collection-player/public-collection-player.component.ts

```diff
@@ -163,6 +164,17 @@ export class PublicCollectionPlayerComponent implements OnInit, OnDestroy {
     }
   }
 
+  public playContentOnClick(content: { title: string, id: string }) {
```

Comment:
Please explain this function as I see this is there to play only content which has a mimeType as xUrl what will happen to other type of content?

Reply:
If the mimeType is xurl, we call the service to get the redirect url, in case of other mimeType this function need not be called